Neo4j.Driver/Neo4j.Driver/V1/Config.cs

```diff
@@ -103,18 +106,41 @@ static Config()
         public ILogger Logger { get; set; } = new DebugLogger {Level = LogLevel.Info};
 
         /// <summary>
-        /// Gets or sets the maximum transaction rety time.
+        /// Gets or sets the maximum transaction rety timeout.
         /// </summary>
         public TimeSpan MaxTransactionRetryTime { get; set; } = TimeSpan.FromSeconds(30);
 
         /// <summary>
-        /// Gets or sets the max idle session pool size.
+        /// Gets or sets the max idle connection pool size.
         /// </summary>
         /// <remarks> 
-        /// The max idle session pool size represents the maximum number of sessions buffered by the driver. 
-        /// A buffered <see cref="ISession"/> is a session that has already been connected to the database instance and doesn't need to re-initialize.
+        /// The max idle connection pool size represents the maximum number of idle connections buffered by the driver. 
+        /// An idle connection is a connection that has already been connected to the database instance and doesn't need to re-initialize.
         /// </remarks>
-        public int MaxIdleSessionPoolSize { get; set; } = 10;
+        public int MaxIdleConnectionPoolSize { get; set; } = 10;
```

Comment:
AFAIK, the equivalent of this in ADO.Net (Max Pool Size) is defaulted to 100. We may consider using that default value later on.